autoPyTorch/datasets/resampling_strategy.py

```diff
@@ -91,10 +91,10 @@ def is_stratified(self) -> bool:
 
 
 # TODO: replace it with another way
-RESAMPLING_STRATEGIES = [CrossValTypes, HoldoutValTypes, NoResamplingStrategyTypes]
+RESAMPLING_STRATEGIES = Union[CrossValTypes, HoldoutValTypes, NoResamplingStrategyTypes]
```

Comment:
Names of typing variables follow the naming of class, so ResamplingStrategies.

Reply:
sure